### PR TITLE
LANG-1474 documentation correction

### PR DESCRIPTION
**Pull request by verhas**

### CODE REVIEW

File: src/main/java/org/apache/commons/lang3/StringUtils.java [hunk 1]
```diff
@@ -3600,7 +3600,7 @@ public static boolean isNoneBlank(final CharSequence... css) {
      * <p>Checks if none of the CharSequences are empty ("") or null.</p>
      *
      * <pre>
-     * StringUtils.isNoneEmpty((String) null)    = false
+     * StringUtils.isNoneEmpty((String) null)    = true
```
Comment (kinow) [inline src/main/java/org/apache/commons/lang3/StringUtils.java]: Same as this one, `false` not `true`.

**Comment by Stzx** [comment]
+1

**Comment by kinow** [comment]
@Stzx do you think the documentation changes are correct, or did you get the same as I did testing it? (sorry, couldn't tell if +1 was for my comment or for the PR change :)

**Comment by Stzx** [comment]
@kinow Sorry.
Make you feel confused.
The conclusion of my test is the same as yours.

**Comment by kinow** [comment]
Thanks for testing, it was my bad too for not understanding it. Let's wait @verhas to comment now :+1:

**Comment by kinow** [comment]
@verhas tested again today, and same behavior in master. Closing the PR, but in case you can confirm it's indeed an issue, feel free to re-open and reply with some step to confirm/reproduce it. Thanks!

**Pull request closed without merging**